cookbooks/bcpc-hadoop/recipes/oozie_pack.rb

```diff
@@ -0,0 +1,98 @@
+#
+# Recipe to include hbase jar files into Oozie
+#
+%w{
+hbase
+}.each do |p|
+  package p do
+    action :install
+  end
+end
+
+file "/tmp/oozie.war" do
```

Comment:
Why do we need to copy the war to a temp directory? We could just unzip the jar from its original location into the temp directory.